CHANGES

```diff
@@ -52,6 +52,16 @@ Vernacular Commands
 - Nested proofs may be enabled through the option `Nested Proofs Allowed`.
   By default, they are disabled and produce an error. The deprecation
   warning which used to occur when using nested proofs has been removed.
+- Added option Uniform Inductive Parameters which abstracts over parameters
+  before typechecking constructors, allowing to write for example
+  `Inductive list (A : Type) := nil : list | cons : A -> list -> list.`
+- Strengthened inference of conclusions of inductive constructors.
```

Comment:
Don't you mean "weakened"?

Reply:
No, inference of the conclusions is definitely stronger with this approach (see tests in https://github.com/jashug/coq/blob/746977132dc591de0ec22510646daf287c0ae3c5/test-suite/bugs/closed/7253.v). As I say below though, inference of the arguments stops using a certain heuristic, which can make it either stronger (when the heuristic was wrong) or weaker (when it was right).